From aebb604b96570cf03b053cb76fe0bd34ce16ded8 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Thu, 02 Apr 2020 01:12:10 +0300
Subject: [PATCH] [SW-11180] preboot: move preboot.h include

Move the preboot.h include to precede the other local and specs
includes. This is essential so Preboot can cope with various decisions
taken in specs.

Change-Id: If2e47ee8c80de915d3ad8a0a9ddd4059c90cb6ae
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/app/preboot.c b/app/preboot.c
index b759c59..a398b27 100644
--- a/app/preboot.c
+++ b/app/preboot.c
@@ -10,11 +10,11 @@
 
 #include <psoc_global_conf_regs.h>
 #include <cpu_ca53_cfg_regs.h>
+#include <preboot.h>
 #include <hl_fw_if.h>
 #include <hl_boot_if.h>
 
 #include "common.h"
-#include "preboot.h"
 #include "uart.h"
 #include "spi.h"
 #include "pll.h"
